performance/nodejs.md

```
@@ -0,0 +1,48 @@
+  for (const user of users) {
+    const posts = await db.query('SELECT * FROM posts WHERE user_id = ?', [user.id]);
+  }
+  ```
+  ```javascript
+  // OK
+  const userIds = users.map(u => u.id);
+  const posts = await db.query('SELECT * FROM posts WHERE user_id IN (?)', [userIds]);
+  ```
+- **対象ファイル例**: `src/services/UserService.js`
+- **参照PR**:
+  - https://github.com/example/repo/pull/345
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)